### PR TITLE
Feature/template tags

### PR DESCRIPTION
**Pull request by Grubba27**
add a template tag feature 
image

closes https://github.com/vinibgoulart/wpp-playground/issues/18

### CODE REVIEW

**Review by akinncar — APPROVED** [review]
nicee

File: src/wweb/onMessage/debug/getGroupIDMessage.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { code } from 'src/utils/templates';
+import { Message } from 'whatsapp-web.js';
+const getGroupID = async (msg: Message) => {
```
Comment (vinibgoulart) [inline src/wweb/onMessage/debug/getGroupIDMessage.ts:3]: groupIdMessage

File: src/wweb/onMessage/debug/getGroupIDMessage.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  try {
+    return msg.reply(code`${msg.id.remote}`);
+  } catch (error) {
+    return msg.reply((error as Error).message);
+  }
+};
+
+export default getGroupID;
```
Comment (vinibgoulart) [inline src/wweb/onMessage/debug/getGroupIDMessage.ts:11]: groupIdMessage

File: src/utils/templates.ts
Comment (vinibgoulart) [inline src/utils/templates.ts:1]: nice

**Pull request merged**